src/backend/joanie/core/utils/contract_definition.py

```diff
@@ -210,3 +218,16 @@ def generate_document_context(contract_definition=None, user=None, order=None):
     }
 
     return apply_contract_definition_context_processors(context)
+
+
+def embed_images_in_context(context):
+    """Embed images in the context."""
+    edited_context = deepcopy(context)
+    try:
+        logo = DocumentImage.objects.get(id=edited_context["organization"]["logo_id"])
+        edited_context["organization"]["logo"] = image_to_base64(logo.file)
```

Comment:
As we are retrieving `logo_id` property now, we have to write a migration to update all existing contexts to prevent to raise a `KeyError` here.

IMO, for fully signed contract, we can assume to just clear the context as the document is stored on the Lex Persona.